tardis/spectrum/base.py

```diff
@@ -0,0 +1,192 @@
+import warnings
+
+import numpy as np
+from astropy import units as u
+
+from tardis.io.util import HDFWriterMixin
+from tardis.spectrum.formal_integral import IntegrationError
+from tardis.spectrum.spectrum import TARDISSpectrum
+from tardis.util.base import (
+    quantity_linspace,
+)
+
+
+class SpectrumSolver(HDFWriterMixin):
+    hdf_properties = [
+        "montecarlo_virtual_luminosity",
```

Comment:
Should we be outputting this?

Reply:
I don't see why not, might at least be useful for debugging